app/views/workflows/_edit_form.erb

```diff
@@ -4,7 +4,7 @@
     <%= f.alert_message "Please fix the errors below." %>
 
     <%= f.text_field :name %>
-    <%= f.select :batch_host, OODClusters.map { |key, val| [ "#{val.title} (#{val.resource_mgr_server.host})", key ] }, { selected: @workflow.batch_host, label: "Batch Server" }, { class: "selectpicker" } %>
+    <%= f.select :batch_host, OODClusters.map { |cluster| [ "#{cluster.metadata_config[:title]} (#{cluster.job_config[:host]})", cluster.id ] }, { selected: @workflow.batch_host, label: "Batch Server" }, { class: "selectpicker" } %>
```

Comment:
This pattern is repeated elsewhere. I can remove the `:host` information, but can we think of a situation where there won't be a submit host for a cluster?